tests/units/anta_tests/test_stp.py

```diff
@@ -324,4 +324,98 @@
         "inputs": {"priority": 32768, "instances": [10, 20, 30]},
         "expected": {"result": "failure", "messages": ["The following instance(s) have the wrong STP root priority configured: ['VL20', 'VL30']"]},
     },
+    {
+        "name": "success-mstp",
+        "test": VerifyStpTopologyChanges,
+        "eos_data": [
+            {
+                "unmappedVlans": [],
+                "topologies": {
+                    "Cist": {
+                        "interfaces": {
+                            "Cpu": {"state": "forwarding", "numChanges": 1, "lastChange": 1723990624.735365},
+                            "Port-Channel5": {"state": "forwarding", "numChanges": 1, "lastChange": 1723990624.7353542},
+                        }
+                    },
+                    "NoStp": {
+                        "interfaces": {
+                            "Cpu": {"state": "forwarding", "numChanges": 1, "lastChange": 1723990624.735365},
+                            "Ethernet1": {"state": "forwarding", "numChanges": 15, "lastChange": 1723990624.7353542},
+                        }
+                    },
+                },
+            },
+        ],
+        "inputs": {"threshold": 10},
+        "expected": {"result": "success"},
+    },
+    {
+        "name": "success-rstp",
```

Comment:
Can you also add a test with `rapid-pvst` please?

Reply:
Added unit test for rapid-pvst.
Thanks!!